voxel51/users/utils.py

```diff
@@ -200,3 +202,26 @@ def _recurse(v):
     elif isinstance(v, Serializable):
         return v.to_dict()
     return v
+
+
+def upload_files(requests, url, files, headers, **kwargs):
+    '''Upload one or more files using a streaming upload.
```

Comment:
Could just say `Uploads one or more files of any size using a streaming upload.` and remove second sentence below.

Reply:
Sounds good. I still want to mention the 2GB bit for context (so someone uploading a 10-byte file knows they don't necessarily need to go this route).